catalog/app/containers/NavBar/Help.js

```diff
@@ -0,0 +1,313 @@
+import cx from 'classnames'
+import * as Lab from '@material-ui/lab'
+import * as M from '@material-ui/core'
+import * as React from 'react'
+
+import StyledLink from 'utils/StyledLink'
+
+const ES_V = '6.7'
+const ES_REF = `https://www.elastic.co/guide/en/elasticsearch/reference/${ES_V}/query-dsl-query-string-query.html#query-string-syntax`
+
+const useStyles = M.makeStyles((t) => ({
+  root: {
+    padding: `0 ${t.spacing(4)}px`,
+    overflowY: 'auto',
+
+    [t.breakpoints.down('xs')]: {
+      padding: `0 ${t.spacing(1)}px`,
+    },
+  },
+  group: {
+    marginTop: t.spacing(2),
+  },
+  headerLabel: {
+    background: 'transparent !important',
+  },
+  item: {
+    borderBottom: `1px solid ${t.palette.divider}`,
+
+    '&:last-child': {
+      border: 0,
+    },
+  },
+  list: {
+    marginBottom: t.spacing(1),
+  },
+  subList: {
+    marginLeft: '-12px',
+  },
+}))
+
+const useCodeStyles = M.makeStyles((t) => ({
+  root: {
+    background: t.palette.grey['300'],
+    color: t.palette.info.contrastText,
+    font: t.typography.monospace,
```

Comment:
this doesnt work (shows `[object Object]` in dev tools), use `t.typography.monospace.fontFamily` instead